test/parallel/test-child-process-env.js

```diff
@@ -48,12 +48,12 @@ let response = '';
 
 child.stdout.setEncoding('utf8');
 
-child.stdout.on('data', function(chunk) {
+child.stdout.on('data', (chunk) => {
   console.log(`stdout: ${chunk}`);
   response += chunk;
 });
 
-process.on('exit', function() {
+process.on('exit', () => {
```

Comment:
Nit: A `common.mustCall` would be better around this. Basically, if there is an assertion in a function, it is better to surround that with `common.mustCall`.

Reply:
Done. Thanks for the heads-up! 👍

Reply:
This should not be wrapped in `common.mustCall()` because `common.mustCall()` depends on `process.on('exit'...)`. By the time you are in the exit handler, it is too late.

Reply:
Thanks for explaining. Reverted the `common.mustCall()` wrapper.